tests/edge.test.ts

```diff
@@ -0,0 +1,18 @@
+import { test } from "@jest/globals"
+import fs from "fs/promises"
+
+import type { EdgeOptions } from "~index"
+import { deployEdge } from "~index"
+
+test("dry run successful", async () => {
+  const opt = JSON.parse(await fs.readFile("keys.json", "utf8"))
+    .edge as EdgeOptions
+
+  await deployEdge({
+    verbose: true,
+    dry: true,
+    cookie: opt.cookie,
+    extId: opt.extId,
+    zip: "test.zip"
```

Comment:
There is a presumption here that a `test.zip` has been created by someone, but there's nothing to facilitate this. Was your intention here that a user would create their own `test.zip`?

Potentially having something like 

```
const testZipExists = await fs.promises.access("test.zip");
if(!testZipExists) {
    fail("test zip not in directory")
}
```

It's helpful to do this so as to separate and make crystal clear the difference of failures within `deployEdge` and failures of the test runner itself due to a user error.

In a perfect world, it would be great if we could actually package a `test.zip` that we know doesn't have any flaws to completely remove any 3rd party failure cases.

Reply:
Yeah this is for unit testing, to be run in an automated manner. We can check if it doesn't exist and download the zip from github release too. Better yet is a way to produce a zip with version always bumping for test purposes.